### DISCUSSION TITLE
Gerrit-Platform autoMerge for consecutive minor/patch updates are not working

### DISCUSSION THREAD

**Discussion by NiasSt90** [renovatebot/renovate#26742] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.140.5

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

The Gerrit `platform.findPr()` don't use the `prTitle` attribute in the search for existing PRs(changes).

Therefore the first minor/patch update to a single dependency will work as expected.
But the next patch/minor update to the same dependency the current Gerrit `findPr()` implementation will find the first and already merged dependency and not the new (automatically) created one. 
Therefore it will never autoMerge this new dependency.

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
Converted to bug issue: https://github.com/renovatebot/renovate/issues/26743

**Reply by NiasSt90** [reply]
thanks, the Platform selector in this "create discussion" dialog is missing Gerrit as a platform.
